onnxruntime/core/providers/dml/DmlExecutionProvider/src/External/D3DX12/d3dx12.h

```diff
@@ -8,7 +8,7 @@
 #ifndef __D3DX12_H__
 #define __D3DX12_H__
 
-#include "d3d12.h"
+#include "C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\d3d12.h"
```

Comment:
That cannot be hardcoded to the fixed SDK version and location of the header file :-) Please remove the absolute path.